xpad.c

```diff
@@ -394,6 +403,13 @@ static const signed short xpad_abs_triggers[] = {
 	-1
 };
 
+/* used when the controller has extra paddle buttons */
+static const signed short xpad_btn_paddles[] = {
+	BTN_TRIGGER_HAPPY5, BTN_TRIGGER_HAPPY6, /* paddle upper right, lower right */
+	BTN_TRIGGER_HAPPY7, BTN_TRIGGER_HAPPY8, /* paddle upper left, lower left */
```

Comment:
I decided to change these to be more consistent with the pattern that SDL was already using. The layout I was using previously was inherited from @Fmstrat's prior work, so I don't necessarily know if there was a compelling reason for it to be the way it was.